### PR TITLE
Tokens create README.md

### PR DESCRIPTION
**Pull request by romellogoodman**
### WHY are these changes introduced?

Creates a README.md for the tokens package

### CODE REVIEW

File: polaris-tokens/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+```js
+import {tokens} from '@shopify/polaris-tokens';
+
+// Access all of the available token groups
+console.log(tokens);
+```
+
+## Migrating from v4 to v5
+
+Polaris Tokens v5.0.0 ([full release notes](https://github.com/Shopify/polaris-react/releases/tag/tokens-v5.0.0)) features an overhaul of the package. The Sass variables, CSS Filters, and Rails have all been removed. The new version is based on json files and utility javascript functions. In order to find your replacements please navigate to [polaris.shopify.com/tokens/getting-started-with-tokens](https://polaris.shopify.com/tokens/getting-started-with-tokens).
```
Comment (alex-page) [inline polaris-tokens/README.md]: We should make a migration document. We have some in `documentation` directory in root.

File: documentation/guides/migrating-tokens-from-v4-to-v5.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## Migrating from v4 to v5
+
+Polaris Tokens v5.0.0 ([full release notes](https://github.com/Shopify/polaris-react/releases/tag/tokens-v5.0.0)) features an overhaul of the package. The Sass variables, CSS Filters, and Rails have all been removed. The new version is based on json files and utility javascript functions. In order to find your replacements please navigate to [polaris.shopify.com/tokens/getting-started-with-tokens](https://polaris.shopify.com/tokens/getting-started-with-tokens).
```
Comment (alex-page) [inline documentation/guides/migrating-tokens-from-v4-to-v5.md:3]: @aaronccasanova @romellogoodman I haven't had a deep look at the tokens work. Should it for now be `json` and `css` output?

Comment (romellogoodman) [reply documentation/guides/migrating-tokens-from-v4-to-v5.md:3]: Want to note that this is more or less just a placeholder. I was going to flush to be more explicit about the removal of the old apis, the new apis, types, and utilities

Comment (alex-page) [reply documentation/guides/migrating-tokens-from-v4-to-v5.md:3]: Oh sweet. Feel free to ignore me then and ship!

**Review by alex-page — APPROVED** [review]

File: documentation/guides/migrating-tokens-from-v4-to-v5.md (hunk 2 above)
Comment (aaronccasanova) [reply documentation/guides/migrating-tokens-from-v4-to-v5.md:3]: Currently, the package exposes all token groups from a `tokens` javascript object and by nature of not using the `pkg.exports` key it also allows access to the individual `JSON` token group files. We don't expose any `CSS` or `SCSS` but do think we should in the near future so perhaps we should exclude that from the documentation.

> but do think we should in the near future

e.g. We currently generate the `CustomProperties` component stylesheet at runtime for every instance. This should probably be generated once during the `polaris-tokens` `yarn build` and simply imported in the `CustomProperties` component (`import '@shopify/polaris-tokens/styles/custom-properties.css'`)

**Pull request merged**